nntrainer/tensor/cl_operations/blas_kernels.h

```diff
@@ -22,7 +22,7 @@
 namespace nntrainer {
 
 // get global cl_context to use in kernels
-extern ClContext &cl_context_ref;
+static ClContext &cl_context_ref = ClContext::Global();
```

Comment:
Don't we need to add `cl_context_ref.initBlasClKernels(); ` here?

Reply:
Similar to what I mentioned above, if users call `initBlasClKernels()`, then the kernels will get registered at the beginning. Otherwise each kernel implementation will separately get registered if its not registered initially. Here `cl_context_ref.initBlasClKernels();` won't be required if users call `initBlasClKernels()` before. If not then individually the kernels will get registered whenever they are called.

Reply:
Hmm.. It sounds `ClContext::Global()` can be called multiple times, and they return different global ClContext right?
I think I missed its flow. I'm gonna take a look at it again.

Reply:
It will return the same static instance. I have slightly updated this line:
`static ClContext &cl_context_ref = ClContext::Global();` -> `static ClContext cl_context_ref;`